casestudy/templates/casestudy/base.html

```diff
@@ -42,7 +42,7 @@ <h1>{{ casestudy.title }}</h1>
                 {% block quote_two_link %}{% endblock %}
             </cite>
         </blockquote>
-        {% include 'core/social-share-links.html' %}
+        {% include 'core/widgets/social-share-links.html' %}
```

Comment:
note the "widgets" folder initially started as a place to put templates for form field widgets.

But I see it's now the widgets folder has also got components to include in other template.

not a problem, just an FYI.